eth/main.cpp

```diff
@@ -965,10 +1029,10 @@ int main(int argc, char** argv)
 			bytes block = web3.ethereum()->blockChain().block(web3.ethereum()->blockChain().numberHash(i));
 			switch (exportFormat)
 			{
-			case Format::Binary: out.write((char const*)block.data(), block.size()); break;
-			case Format::Hex: out << toHex(block) << "\n"; break;
-			case Format::Human: out << RLP(block) << "\n"; break;
-			default:;
+				case Format::Binary: out.write((char const*)block.data(), block.size()); break;
```

Comment:
`case` starts on the same column as braces, don't change the indentation here and in other `switch`es